BrainPortal/app/models/bourreau.rb

```diff
@@ -31,6 +31,8 @@ class Bourreau < RemoteResource
 
   attr_accessor :operation_messages # no need to store in DB
 
+  api_attr_visible :id, :name, :user_id, :group_id, :online, :read_only, :description
```

Comment:
the attribute `:id` is ALAWAYS visible, it's configured as such automatically by api_attr_visible, so there' sno need to specify it every time.